script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -105,12 +106,12 @@ A deep red stain has blossomed on their white shirt.
 [sha:f9747db87335eda9fea82f751cd52f6854719632]{
 -- Page 5, Offset 28991.
 -- それを見下ろしている。
--- TRANSLATION HERE
+I am looking down upon them.
```

Comment:
is this "looking down from an elevated position"? I thought this was "looking down" in the metaphorical sense?

Reply:
I think it's physical, he's standing